contrib/zpages/src/main/java/io/opencensus/contrib/zpages/StatszZPageHandler.java

```diff
@@ -0,0 +1,558 @@
+    for (int i = 0; i < bucketCounts.size(); i++) {
+      double low = i == 0 ? Double.NEGATIVE_INFINITY : bucketBoundaries.get(i - 1);
+      double high =
+          i == bucketCounts.size() - 1 ? Double.POSITIVE_INFINITY : bucketBoundaries.get(i);
+      out.write("<tr>");
+      formatter.format("<td>[%.3f...%.3f)</td>", low, high);
+      formatter.format("<td>%d</td>", bucketCounts.get(i));
+      out.write("</tr>");
+    }
+    out.write("</tbody>");
+    out.write("</table>");
+    out.write("</td>");
+  }
+
+  private static void emitMeasureTable(
+      Map<String, Measure> measures, PrintWriter out, Formatter formatter) {
+    out.write("<h2>Measures</h2>");
+    out.write("<p></p>");
+    formatter.format("<table %s frame=box cellspacing=0 cellpadding=2>", TABLE_BORDER);
+    emitMeasureTableHeader(out, formatter);
+    out.write("<tbody>");
+    for (Entry<String, Measure> entry : measures.entrySet()) {
+      emitMeasureTableRow(entry.getValue(), out, formatter);
+    }
+    out.write("</tbody>");
+    out.write("</table>");
+    out.write("<p></p>");
+  }
+
+  private static void emitMeasureTableHeader(PrintWriter out, Formatter formatter) {
+    out.write("<thead>");
+    out.write("<tr>");
+    formatter.format("<th colspan=1 class=\"l1\">%s</th>", TABLE_HEADER_MEASURE);
+    formatter.format("<th colspan=1 class=\"l1\">%s</th>", TABLE_HEADER_DESCRIPTION);
+    formatter.format("<th colspan=1 class=\"l1\">%s</th>", TABLE_HEADER_UNIT);
+    formatter.format("<th colspan=1 class=\"l1\">%s</th>", TABLE_HEADER_MEASURE_TYPE);
+    out.write("</tr>");
+    out.write("</thead>");
+  }
+
+  private static void emitMeasureTableRow(Measure measure, PrintWriter out, Formatter formatter) {
+    out.write("<tr>");
+    formatter.format("<td><b>%s</b></td>", measure.getName());
+    formatter.format("<td align=\"left\">%s&nbsp;</td>", measure.getDescription());
+    formatter.format("<td align=\"left\">%s&nbsp;</td>", measure.getUnit());
+    String measureType =
+        measure.match(
+            Functions.returnConstant("Double"),
+            Functions.returnConstant("Long"),
+            Functions.throwAssertionError());
+    formatter.format("<td align=\"left\">%s&nbsp;</td>", measureType);
+    out.write("</tr>");
+  }
+
+  static StatszZPageHandler create(ViewManager viewManager) {
+    return new StatszZPageHandler(viewManager);
+  }
+
+  private StatszZPageHandler(ViewManager viewManager) {
+    this.viewManager = viewManager;
+  }
+
+  /*
+   * TreeNode for storing the structure of views and directories that they're in. Think of this as
+   * file descriptors for view: non-leaf nodes are directories which may contain views or other
+   * directories, and leaf nodes are the ones with actual information on views. Each non-leaf node
+   * also has the number of views under its directory.
+   */
+  private static class TreeNode {
+    // Only leaf nodes have views.
+    @javax.annotation.Nullable final /*@Nullable*/ View view;
```

Comment:
The commented annotation is redundant.

Reply:
Created https://github.com/census-instrumentation/opencensus-java/pull/1005 to fix this.